packages/d3fc-chart/README.md

```diff
@@ -78,33 +78,167 @@ The chart is constructed using a pair of scales. The scale configuration propert
 
 ### Styles
 
-The component uses inline style attributes to apply the necessary CSS to the elements. To override these values, use the decorate functionality -
+The component automatically injects a stylesheet to apply the necessary CSS to the elements. These styles can be augmented with CSS -
+
+```css
+d3fc-group.cartesian-chart > .x-label {
+  line-height: 5em;
+  color: red;
+}
+```
+
+Or through decoration -
 
 ```javascript
 chart.decorate(selection => {
   selection.enter()
-      .select('.chart-label')
-      .style('height', '5em')
-      .style('line-height', '5em');
+    .select('.x-label')
+    .style('line-height', '5em')
+    .style('color', 'red')
+});
+```
+
+### Grid
+
+Internally, the component uses CSS grid layout to arrange the various rendering surfaces and labels. The following diagram shows the grid structure and associated line indicies -
+
+```
+  1             2           3               4            5              6
+1 /-------------|-----------|---------------|------------|--------------\
+  |             |           |   top gutter  |            |              |
+2 |-------------|-----------|---------------|------------|--------------|
+  |             |           |    top axis   |            |              |
+3 |-------------|-----------|---------------|------------|--------------|
+  | left gutter | left axis |   plot area   | right axis | right gutter |
+4 |-------------|-----------|---------------|------------|--------------|
+  |             |           |  bottom axis  |            |              |
+5 |-------------|-----------|---------------|------------|--------------|
+  |             |           | bottom gutter |            |              |
+6 \-------------|-----------|---------------|------------|--------------/
+```
+
+Additional elements can be dropped into this grid, either in-place of or in addition to the existing elements using decoration. The following are some typical examples of this process.
+
+N.B. Each of these examples uses the modern CSS grid specification syntax. To make these work in legacy versions of IE, you'll need to additionally set the following equivalent prefixed properties -
+
+* `grid-column` - `-ms-grid-column`
+* `grid-row` - `-ms-grid-row`
+
+#### Add a Chart Label
+
+There's no configuration option to add a title to the chart itself but it can be easily added using decoration -
+
+```javascript
+chart.decorate(selection => {
+  // select the top-label (if it exists)
+  selection.select('.top-label')
+    // move the label down to make space for the chart label
+    .style('margin-top', '2em');
+
+  // when the chart is added to the DOM
+  selection.enter()
+    // additionally add a div element for the label
+    .append('div')
+    .attr('class', 'chart-label')
+    // move the element into the top-label cell
+    .style('grid-column', 3)
+    .style('grid-row', 1)
+    // add some styling to the label
```

Comment:
as you're not including a screenshot, perhaps remove the lines relating to styling for the sake of brevity?